docker-compose.yml

```diff
@@ -43,6 +46,7 @@ services:
       - AIRFLOW_RESULT_URL=redis://redis:6379/0
       - AIRFLOW_FLOWER_PORT=5555
       - AIRFLOW_DATABASE_URL=mysql://root:secret@db:3306/airflow
+      - AIRFLOW__CORE__DAGS_FOLDER=/app/devdags
```

Comment:
This breaks the `test-parse` CI, since the devdags folder is empty. We do want to test against all of the existing dags to make sure we're not making bad commits.

Reply:
rm'd.